src/porepy/geometry/half_space.py

```diff
@@ -132,27 +119,27 @@ def half_space_interior_point(
 def vertexes_of_convex_domain(A: np.ndarray, b: np.ndarray) -> np.ndarray:
     """Find the vertexes of a convex domain specified as an intersection of half spaces.
 
-    The function assumes the domain is defined by inequalities on the form
-
-        A * x + b <= 0
-
-    For more information, see scipy.spatial functions HalfspaceIntersection.
+    Note:     
+        The function assumes the domain is defined by inequalities on the form
 
-    The function has been tested for 2d and 3d domains.
+            A * x + b <= 0
 
-    Error messages from HalfspaceIntersection are quite possibly from qhull.
-    These will often reflect errors in the way the matrix A and b are set up
-    (e.g. sign errors that imply that the inequalities do not form a closed domain).
+        For more information, see scipy.spatial functions HalfspaceIntersection.
 
-    Parameters
-    ----------
-    A : np.ndarray, size num_planes x num_dim
-        Matrix of normal vectors (in rows) for the half planes. Should be oriented
-        so that A * x + b < 0
-    b : np.ndarray, size num_planes
-        Constants used to define inequalities of the half spaces. Should be scaled
-        so that A * x + b < 0.
+        The function has been tested for 2d and 3d domains.
 
+    Parameters:
+        A (shape = (num_planes, num_dim)): Matrix of normal vectors (in rows) for the half planes. Should be oriented
+            so that A * x + b < 0
+        b (len = num_planes): Constants used to define inequalities of the half spaces. Should be scaled
+            so that A * x + b < 0.
+            
+    Returns:
+        Vertexes of a convex domain 
+    
+    Raises: 
+        QhullError: If A and b are not set up right (e.g. sign errors that imply that the inequalities do not form a closed domain).     
```

Comment:
```suggestion
        QhullError: If ``A`` and ``b`` are not set up right (e.g. sign errors that imply that the inequalities do not form a closed domain).     
```

Reply:
In your latest commit, `QhullError` is written twice. Could you please fix this?